packages/typespec-go/test/armapicenter/fake/zz_apidefinitions_server.go

```diff
@@ -18,60 +18,60 @@ import (
 	"regexp"
 )
 
-// ApiDefinitionsServer is a fake server for instances of the armapicenter.ApiDefinitionsClient type.
-type ApiDefinitionsServer struct {
-	// CreateOrUpdate is the fake for method ApiDefinitionsClient.CreateOrUpdate
+// APIDefinitionsServer is a fake server for instances of the armapicenter.APIDefinitionsClient type.
+type APIDefinitionsServer struct {
```

Comment:
Do we capitalize `API` everywhere? API almost always is followed by a capitilized object. Thus API almost always shows up as APIA, APIB, APIC, APID, .....

Reply:
Yes, acronyms are always capitalized.